site/_posts/2017-10-16-how-was-your-last-week.md

```diff
@@ -0,0 +1,166 @@
+
+Yes. Me and [himdel] are teaching students on some university in Brno. Last week we had a lesson where we teach them about git and
+creaing pull requests.
+
+**How it is related to open source?**
+
+We are showing them our open source project and we are apprising them with for example with code reviews. We are trying 
+to show them more practical view on open source development. Most likely some of them will became contributors of ManageIQ :)
+
+Very good, thanks Zita! 
+
+---
+
+**Hi [Hyperkid123], tell me something about yourself first.**
+
+I am working in UI team and I am intern.
+
+**How was your last week in ManageIQ ?**
+
+I was working topology graphs.
+
+**What did you do with them?**
+
+I was converting them from SVG to CANVAS. Also I am encapsulating them to standalone angular component.
+
+**Is it good ?**
+
+Yes, certainly. You can you can one angular component instead of some angular controller. You can also test it nicely.
+
+**Will it change also visual page ?**
+
+Yes, look on it:
+
+![New Topology](/assets/images/blog/new-topology.png)
+
+Thanks Martin, good news! 
+
+---
+
+**Hi [skateman],
+how was your last week ?**
+
+I did some fixes for our trees. After the conversion to Angular some regressions appeared.
+
+**It is nice that we are using angular components more and more.**
+
+Yes I was also creating tests for angular components as well. Y'know, tests in javascript.
+
+**Do you like it?**
+
+No.
+
+**:D**
+
+**What else ?**
+
+I was preparing a demo for [Open Source Summit Europe][summit].
+
+**What will be demo about?**
+
+We will show ManageIQ to the attendees in a containerized environment.
+
+**How ?**
+
+We have an oVirt cluster on a set of Intel NUC boxes up and running now I'm deploying some VMs to run OpenShift. 
+ManageIQ will be running as a container.
+
+![NUC with ManageIQ](/assets/images/blog/nuc.jpg)
+
+Very good. Thanks David!
+
+---
+
+**Hi [martinpovolny],
+how was your last week ?**
+
+I was mostly doing PR reviews and merging PRs.
+
+**Can you highlight on some of them ?**
+
+Yes. [Aparna] did lot of work on generic objects UI.
+
+[Milan] was simplifying i18 strings. It will help the translators.
+
+[himdel] did lot of toolbar button send checked items clean up.
+
+Thanks Martin for answering also!
+
+---
+
+<outro>
+
+Hopefully, you had also exciting last week as we have had. 
+But if no, you can always took any of our [issues](https://github.com/ManageIQ/manageiq/issues)
+and create PR !
+
+:)
+
+
+[lpichler]: https://github.com/lpichler
+[lsmola]: https://github.com/Ladas
+[aws_test]: https://github.com/ManageIQ/manageiq-providers-amazon/pull/312
+[znemecko]: https://github.com/ZitaNemeckova
+[himdel]: https://github.com/himdel
+[Hyperkid123]: https://github.com/Hyperkid123
+[skateman]: https://github.com/skateman
+[summit]: http://events.linuxfoundation.org/events/open-source-summit-europe
+[martinpovolny]: https://github.com/martinpovolny
+[AparnaKarve]: https://github.com/AparnaKarve
```

Comment:
This should just be [Aparna], as that is what you reference on line 137

Reply:
This is totally up to you,..but I like knowing the names of the people you reference. (I have no idea who Hyperkid123 is?)  so perhaps use their first names [Libor] [Zita] [MartinP]    but I leave it to you. :)

Reply:
yes I agree, I was using GitHub logins and first names - so I unified it - now we have only first names.